code/__DEFINES/role_preferences.dm

```diff
@@ -116,6 +118,7 @@ GLOBAL_LIST_INIT(special_roles, list(
 	ROLE_CHANGELING = 0,
 	ROLE_CLOWN_OPERATIVE = 14,
 	ROLE_CULTIST = 14,
+	ROLE_FAMILIES = 0, // SKYRAT EDIT
```

Comment:
put this with assops

Reply:
they were placed in that spot to preserve their original location in the UI but I can move them